README.md

```diff
@@ -847,6 +847,47 @@ Options for status only:
 Retrieve the object sampling status, or reset it.
 Project is a required flag if fingerprint is specified.
 
+#### Configuring Sampling (Coronerd 1.50+)
+
+In Coronerd 1.50, it became possible to configure sampling on a per-project
+basis as well as using `coronerd.conf`.  This is done with the
+`morgue sampling configure` command. Configurations specified on projects
+override the `coronerd.conf` settings.  For example:
+
+```
+morgue sampling configure --project myproject \
+--attribute version \
+--backoff 1,0 \
```

Comment:
we should explain what the command means after giving out this example.

Reply:
We can't without explaining the entire sampling algorithm here because our sampling algorithm isn't simple enough for this readme. I will find out if we have documentation to link yet, but I don't think we do.